packages/design-system/src/components/N8nMarkdown/Markdown.vue

```diff
@@ -247,11 +247,7 @@ export default defineComponent({
 	}
 
 	img {
-		width: 100%;
-		max-height: 90vh;
-		object-fit: cover;
-		border: var(--border-width-base) var(--color-foreground-base) var(--border-style-base);
-		border-radius: var(--border-radius-large);
+		max-width: -webkit-fill-available;
```

Comment:
This won't work in Firefox. Can we use `100%` here?

Reply:
thanks for catching this.. was on my list to check but forgot..